content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+---
+title: Prometheus and OpenTelemetry: Better Together
+linkTitle: Prometheus and OpenTelemetry
+date: 2024-03-10
+author: 
+  [Reese Lee](https://github.com/reese-lee) (New Relic),
+  [Adriana Villela](https://github.com/avillela) (ServiceNow)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # This will be added in future
+---
+
+![Image of a Greek god holding a torch with the Prometheus logo, and OTel logo](Prom-and-otel-logos.png)
+
+Tools such as [Prometheus](https://prometheus.io/) and [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) help us maintain the health, performance, and 
+availability of our complex distributed systems. Both are open source projects 
+under the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) 
+umbrella – but what role does each play in observability? 
+
+OpenTelemetry (OTel for short), is a vendor-neutral open standard for 
+instrumenting, generating, collecting, and exporting telemetry data. Prometheus 
+is a fixture of the observability landscape, widely relied upon for monitoring 
+and alerting within organizations. 
+
+While both Prometheus and OTel emit metrics, there is a lot to cover on the 
+differences and similarities, and is out of scope for this article. Rather, we 
+want to show you how OTel supports Prometheus, specifically in a Kubernetes 
+environment. You will learn:
+* How the OTel Collector's Prometheus Receiver can be used to ingest Prometheus 
+metrics.
+* How the OTel Operator's Target Allocator (TA) can be used for Prometheus 
+service discovery. 
+* How the TA ensures even distribution of Prometheus targets. 
+* Alternative methods for Prometheus metric collection through OTel-native 
+options such as the K8s cluster receiver and Kubelet stats receiver.  
+
+## Prometheus
+Prometheus encompasses many things, including [a time-series data store, 
+monitoring and alerting tool](https://prometheus.io/), and [data format](https://grafana.com/docs/grafana/latest/fundamentals/intro-to-prometheus/), known as [Prometheus 
+text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats).
+
+Prometheus _data_ is stored as a dimensional time-series, meaning that the data 
+has attributes (i.e. labels/dimensions) and a timestamp. 
+
+The Prometheus _tool_ collects Prometheus metrics data from targets defined in a 
```

Comment:
Server sounds more precise here, it's also the term the [Prometheus docs](https://prometheus.io/docs/introduction/overview/) themselves use.
```suggestion
The Prometheus _server_ collects Prometheus metrics data from targets defined in a 
```